scenarios/sns_secrets/cheat_sheet.md

```diff
@@ -0,0 +1,153 @@
+# SNS_Secrets Walkthrough
+
+## Summary
+
+In this scenario, you are provided with AWS credentials. You need to enumerate permissions and discover you can list and subscribe to SNS topics. You then use Pacu's new modules "sns__enum" and "sns__subscribe" to subscribe to the topic. You will receive an email from the topic with an API Key as a debug message. Then, use the AWS CLI to enumerate the API Gateways to find the API Gateway path, method, stage, and resource. Finally, do a curl request with the API key to get the final flag.
+
+## Detailed Walkthrough
+
+### Configuring the AWS Keys
+
+After launching the scenario, you will be provided with an Access Key ID and Secret Access Key for an IAM role. These allow you to interact with the AWS account from the IAM role using the CLI (Command Line Interface). You can set up an IAM profile with this command:
+
+```bash
+aws configure --profile sns-secrets
+```
+
+You will then be prompted for the Access Key ID, Secret, and Region. Provide the keys you received after launching the scenario. Below is an example (your keys will be different.)  
+
+```bash
+AWS Access Key ID: AKIA2GIZ2CSU6NO2YIXU
```

Comment:
Could get your account id from this, but its probably already been leaked 😄 

```bash
aws sts get-access-key-info --access-key-id AKIA2GIZ2CSU6NO2YIXU
```

Reply:
Haha yeah, this AWS account is ONLY used as a sandbox and live stream (it's a personal sandbox of mine). The account id is all over my live streams do no worries there :D